spk/fuse-exfat/Makefile

```diff
@@ -0,0 +1,24 @@
+SPK_NAME = fuse-exfat
+SPK_VERS = 1.2.1
+SPK_REV = 3
```

Comment:
Start with `1`. It only goes up when we plan to release a new version of the SPK.